src/graphql/main/schema.graphql

```diff
@@ -1346,6 +1354,11 @@ type User {
   contacts: [UserContact!]! @deprecated(reason: "will be moved to account")
   createdAt: Timestamp!
   defaultAccount: Account!
+
+  """
+  Device ID of the user's device.
+  """
+  device: String
```

Comment:
how about `deviceId` as the variable name?

Reply:
yep, updated to use deviceId